lib/data/wrangler.py

```diff
@@ -17,8 +17,7 @@
 
 def readData(fileLoc=None):
     data = np.load(fileLoc)
-    print(data.keys())
-    return (data[item] for item in data.keys())
+    return data
```

Comment:
To make this dynamic(actually just fewer changes in case of changing names of `Xtrain`,` ytrain`, etc) by making `app.py` not reliant on the keys we could return a tuple of keys along with the `data` that way if we choose to change `XTrain` to `X_train` or `Xtrain` it'll still read the data.
So, return would be something like `return data, xtrain_key, xtest_key, y_train_key, y_test_key ` and when called in `app.py` we have `items = readata()` `data = items[0], XTrain = data[items[1]], ...  `